lib/api/objectGet.js

```diff
@@ -187,12 +197,29 @@ function objectGet(authInfo, request, returnTagCount, log, callback) {
                 dataLocator = setPartRanges(dataLocator, byteRange);
             }
         }
+        if (awsGetPreCheck) {
```

Comment:
let's move this out of here...

Reply:
as discussed, I updated to check if the `dataStoreType` is an external backend instead and use a multipleBackendsGateway.head method